.gitignore

```diff
@@ -14,3 +14,4 @@ secret*
 *.pyc
 /.vagrant
 .nyc_output
+npm-debug.log
```

Comment:
Depends on your definition of better. Better for me, maybe. Better for the project as a whole, I'm not so sure.

I tend to see global git ignore as good for system-wide stuff. `.DS_Store`, I'm looking at you. But the project-level ignore file should cover everything pursuant to the nature of the project imho, so that potential future contributors aren't all forced to make the same mistake before updating their own global ignore file.

But having said all that, if it's a problem to include it here, I'm obviously happy to remove it. Should I take it out?